app/src/androidTest/java/com/epfl/drawyourpath/pathDrawing/PathDrawingEndFragmentTest.kt

```diff
@@ -46,12 +47,13 @@ class PathDrawingEndFragmentTest {
     fun checkThatClickingOnBackMenuButtonShowMainAppActivity() {
         val intent = Intent(
             ApplicationProvider.getApplicationContext(),
-            PathDrawingActivity::class.java,
+            MainActivity::class.java,
         )
-        intent.putExtra(PathDrawingActivity.EXTRA_COUNTDOWN_DURATION, 1L)
-        val t: ActivityScenario<PathDrawingActivity> = ActivityScenario.launch(intent)
+        intent.putExtra(MainActivity.EXTRA_DRAW_TEST, true)
+        val t: ActivityScenario<MainActivity> = ActivityScenario.launch(intent)
+
         // wait that the countdown passed
-        Thread.sleep(1001)
+        Thread.sleep(4010)
         // click on pause button
```

Comment:
Maybe it could be a good idea to add an extra for the countdown duration (like in the countdown fragment test), zo decrease the duration of the test like in the previous implementation of this test.